grid/project-management/tasks/profile.supr

```diff
@@ -0,0 +1,71 @@
+"""
+Manage tasks in project management systems.
+"""
+name = "project-management/tasks"
+version = "1.0.0"
+
+"""
+Create a new task.
+"""
+usecase CreateTask {
+	input {
+
+    """
+    projects
+    An array of IDs of projects this task belongs to.
+    """
+    projects! [string]!
```

Comment:
I think it would be safer to start with single project, ie.

```suggestion
    project! string!
```

...assuming most PM systems are rather hierarchical (just when I consider Trello or GitHub issues). Asana's multihoming is rather specific feature.

Reply:
I thought that in that case, the map may just create multiple issues in different projects? Though I do see how the semantics differ now, though.